test-suite/src/dictionary_index.rs

```diff
@@ -0,0 +1,21 @@
+use {crate::*, gluesql_core::prelude::Value::*};
+
+test_case!(ditionary_index, async move {
+    run!("CREATE TABLE Foo (id INT PRIMARY KEY, name TEXT);");
```

Comment:
There exists only a single line which is not covered.
ref.
https://coveralls.io/builds/53336746/source?filename=core%2Fsrc%2Fexecutor%2Ffetch.rs#L220

Could we fill the last one?
In here, if we add another table which does not have PRIMARY KEY before adding `Foo` and run `SELECT * FROM GLUE_INDEXES`, then we can expect the test to reach the line.